junit/src/main/java/net/fabricmc/loader/impl/junit/FabricLoaderLauncherSessionListener.java

```diff
@@ -59,4 +72,37 @@ public void launcherSessionClosed(LauncherSession session) {
 		final Thread currentThread = Thread.currentThread();
 		currentThread.setContextClassLoader(launcherSessionClassLoader);
 	}
+
+	// Read system properties from a "fabric-loader-junit.properties" file on the classpath.
+	private static void readClasspathSystemProperties() {
+		try (InputStream is = FabricLoaderLauncherSessionListener.class.getClassLoader().getResourceAsStream(PROPERTY_FILE_NAME)) {
+			if (is == null) {
+				return;
+			}
+
+			try (BufferedReader reader = new BufferedReader(new InputStreamReader(is, StandardCharsets.UTF_8))) {
```

Comment:
Can't this just use `java.util.Properties`, which would also fully support the .properties format?